src/GeneratorTwigHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ *
+ * @internal
```

Comment:
This one is actually *not* internal, as we're exposing it inside of the templates as a variable